packages/rollout/tsconfig.lib.json

```
@@ -1,18 +1,13 @@
 {
-    "compileOnSave": false,
+    "extends": "./tsconfig.json",
```

missed extend + related changes
```suggestion
    "extends": "../../tsconfig.json",
```

---

used correct default config